src/semgrep_agent/main.py

```diff
@@ -72,6 +72,7 @@ def main(
     )
 
     # Get Metadata
+    server = os.getenv("INPUT_PUBLISHURL") or "https://semgrep.dev"
```

Comment:
The click library gives us this value in the `publish_url` function parameter. That one is actually better, cause env vars are not the only way to override the URL, you could also use `--publish-url` on the command line.

So I'd just use the `publish_url` variable later instead of setting `server`.